velox/docs/functions/spark/window.rst

```diff
@@ -0,0 +1,14 @@
+================
+Window functions
+================
+
+Spark window functions can be used to compute SQL window functions.
+
+Value functions
+_______________
```

Comment:
Can you make sure this page shows up correctly in https://facebookincubator.github.io/velox/spark_functions.html?  
Also this line should be dashes instead of underlines.

Reply:
Sure.

Reply:
Looks coverage util now doesn't support window functions. I added the window function support back, and updated spark_functions.rst and coverage.rst.
Thanks.